sdk/java/pulumi/src/test/java/com/MyComponent.java

```diff
@@ -0,0 +1,13 @@
+package com;
```

Comment:
Not sure if there's a better way to add these test classes other than checking these in?  At least it's only in `src/test` so won't end up in the real package, so maybe it's okay?